src/libs/CurrencyUtils.ts

```diff
@@ -122,6 +122,24 @@ function convertToDisplayString(amountInCents = 0, currency: string = CONST.CURR
     });
 }
 
+/**
+ * Given the amount in the "cents", convert it to a short string (no decimals) for display in the UI.
+ * The backend always handle things in "cents" (subunit equal to 1/100)
+ *
+ * @param amountInCents – should be an integer. Anything after a decimal place will be dropped.
+ * @param currency - IOU currency
+ */
+function convertToShortDisplayString(amountInCents = 0, currency: string = CONST.CURRENCY.USD): string {
```

Comment:
Should the currency be of type `string`? Maybe it should be of type `PreferredCurrency` or `ValueOf<typeof CONST.CURRENCY>`?

Reply:
I wanted it to be as reusable as the other currency related util functions! `PreferredCurrency` only includes the currencies supported by Payment card (4 to be exact) and there are more currencies than that, so this type would be too narrow. Hope that answers your question!